public/resources/core/js/controllers/product-offering.controller.js

```diff
@@ -362,6 +423,7 @@
                 vm.createPricePlanStatus = DATA_STATUS.LOADED;
                 vm.pricePlan = new Offering.PricePlan();
                 vm.pricePlanEnabled = false;
+                vm.priceAlteration = false;
```

Comment:
Why are you assigning a false value to something which  have a integer value in the rest of the code